pages/osx/setfile.md

```diff
@@ -0,0 +1,16 @@
+# setfile
+
+> Sets file attributes on files in an HFS+ directory.
+> More information: <https://ss64.com/osx/setfile.html>.
+
+- Set creation date for file:
+
+`setfile -d "{{MM/DD/YYYY HH:MM:SS}}" {{path/to/file}}`
```

Comment:
```suggestion
- Set creation date for specific files:

`setfile -d "{{MM/DD/YYYY HH:MM:SS}}" {{path/to/file1 path/to/file2 ...}}`
```

Optional fix: `a specific` may sound better.
Optional fix: [option specification](https://www.manpagez.com/man/1/SetFile/) violation: multiple files can be passed.